packages/css/src/components/card/card.scss

```diff
@@ -94,6 +94,10 @@
     align-items: center;
     display: flex;
     text-align: center;
+
+    p {
```

Comment:
ok, dit was precies de wens. maar... wat nu als er 2 p's staan, dan wil je daar toch wel misschien weer ruimte tussen? of is dat in zo'n `dso-card-interaction` nooit aan de orde?

Reply:
Interessante vraag, er kunnen wel meerdere `interactions` staan, maar die krijgen allemaal een eigen `dso-card-interaction`. Ik verwacht niet dat er meerdere p's in deze wrapper komen

Reply:
ok